<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D250</class>
 <widget class="QDialog" name="D250">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>956</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Button_Close">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Data_Number">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Folders">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Folders">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Folders</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Data_Number">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Stage_Parameter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stage Parameters</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="Button_MOVE">
      <property name="text">
       <string>MOVE</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Button_GoORIGIN">
      <property name="text">
       <string>Go ORIGIN</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Button_SetORIGIN">
      <property name="text">
       <string>Define ORIGIN</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="LineEdit_StageX"/>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEdit_StageY"/>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEdit_StageZ"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>691</width>
     <height>941</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="PlotWidget" name="graphicsView3"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="PlotWidget" name="graphicsView2"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="PlotWidget" name="graphicsView1"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Label_Stage_X">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X (um)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Stage_Y">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y (um)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Stage_Z">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z (um)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Folder">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Measurement_Parameter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Measurement Parameters</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="LineEdit_Stage_LSpeed"/>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEdit_Stage_FSpeed"/>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEdit_Stage_MicroStep">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>421</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLineEdit" name="LineEdit_Speed"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Response"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Accuracy">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_XFrom"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_XTo"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_Xpitch"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_XTime">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_YFrom"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_YTo"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_Ypitch"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_YTime">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_ZFrom"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_ZTo"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_Zpitch"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_Knife_ZTime">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>480</y>
     <width>95</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="Button_Knife_Measure">
      <property name="text">
       <string>Measure</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Label_ZSCAN_Parameter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ZSCAN Parameters</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>565</y>
     <width>421</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLineEdit" name="LineEdit_ZSCAN_From"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_ZSCAN_To"/>
    </item>
    <item>
     <widget class="QLineEdit" name="LineEdit_ZSCAN_pitch"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>595</y>
     <width>95</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QPushButton" name="Button_ZSCAN_Measure">
      <property name="text">
       <string>Measure</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Label_Knife_From">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>From (um)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Stage_LSpeed">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L Speed (um/s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Stage_FSpeed">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>F Speed (um/s) </string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Stage_MicroStep">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Micro step</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Knife_X">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Knife_Y">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Knife_Z">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Knife_To">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To (um)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Knife_pitch">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pitch (um)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Knife_Time">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time (s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Measure_Speed">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed (um/s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Measure_Response">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Response (ms)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Measure_Accuracy">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Accuracy (us)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_ZSCAN_From">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From (um)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_ZSCAN_To">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>550</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To (um)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Knife_Parameter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>KnifeEdge Parameters</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_ZSCAN_pitch">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>550</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pitch (um)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Position">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>139</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Position</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboBox_Direction">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>485</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CCW</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label_X_Direction">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>485</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X Direction</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_diameter_estimate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>485</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Diameter_Estimate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>485</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Scan_Number">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>600</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Scan_Number">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>600</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan Number</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Button_Close</sender>
   <signal>clicked()</signal>
   <receiver>D250</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
